localstack.go

```diff
@@ -84,8 +84,10 @@ func (s *Stack) stop() error {
 	if !s.started || s.containerID == "" {
 		return nil
 	}
-	timeout := time.Second
-	if err := s.cli.ContainerStop(context.Background(), s.containerID, &timeout); err != nil {
+	timeout := 10 //seconds
```

Comment:
How about renaming the variable to `timeoutInSecond`? Then you can get rid of the comment.

Reply:
Sure